frontend/src/components/ForgotPassword.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <form @submit.prevent="sendEmail">
+    <h3>Forgot password</h3>
+      <div class="form-group">
+        <label>Email</label>
+        <input type="email" name="email" class ="form-control" v-model="email" placeholder="Email"/>
```

Comment:
En aquest input jo afegiria l'atribut "required", que en cas contrari es permet enviar el formulari buit. En enviar-ho buit el server retorna un "422 Unprocessable entry" amb cos "The recipient address is empty", però tot i així es procedeix a la pàgina de reset. Afegint aquest atribut crec que s'arreglaria.

```suggestion
        <input type="email" name="email" class ="form-control" v-model="email" placeholder="Email" required/>
```